main.go

```diff
@@ -155,3 +155,16 @@ func toTable(tablePkg *types.Package, annotationComment string, gd *ast.GenDecl,
 
 	return table, nil
 }
+
+func trimAnnotation(comment string) string {
+	if trimmed := strings.TrimPrefix(comment, "//+table: "); trimmed != comment {
+		return trimmed
+	}
+	if trimmed := strings.TrimPrefix(comment, "// +table: "); trimmed != comment {
+		return trimmed
+	}
+	if trimmed := strings.TrimPrefix(comment, "//sqlla:table "); trimmed != comment {
+		return trimmed
+	}
+	return comment
```

Comment:
one of prefix cases comparing to a comment is better; like table driven test.

```suggestion
	prefixies := []string{"//+table: ", "// +table: ", "//sqlla:table "}
	for _, prefix := range prefixies {
		if trimmed := strings.TrimPrefix(comment, prefix); trimmed != comment {
			return trimmed
		}
	}
	return comment
```

Reply:
:+1: